onoff.js

```diff
@@ -5,8 +5,13 @@ const debounce = require('lodash.debounce');
 const Epoll = require('epoll').Epoll;
 
 const GPIO_ROOT_PATH = '/sys/class/gpio/';
+
+// fs reads return char data
 const ZERO = Buffer.from('0');
 const ONE = Buffer.from('1');
+// lib returns numeric data
+const GPIO_ONE = 1;
+const GPIO_ZERO = 0;
```

Comment:
It would be better to call the variable  `LOW` rather than `GPIO_ZERO` here. Most other libraries tend to use `LOW` too so it will be more familiar to people and therefore easier to understand.